tests/utils/function.test.js

```diff
@@ -0,0 +1,14 @@
+/* global assert, suite, test */
+var bind = require('utils/function').bind;
+
+suite('utils.function', function () {
+  test('utils.function.bind binds to correct object', function () {
+    var obj = {
+      propName: 'aframe',
+      getProp: function () {
+        return this.propName;
+      }
+    };
+    assert.equal(obj.getProp(), bind(obj.getProp, obj)());
```

Comment:
Would it be useful to have a test that binds to a different `obj` with different value?